pvlib/iotools/pvgis.py

```diff
@@ -516,17 +516,16 @@ def get_pvgis_tmy(latitude, longitude, outputformat='json', usehorizon=True,
     elif outputformat == 'csv':
         with io.BytesIO(res.content) as src:
             data, months_selected, inputs, meta = _parse_pvgis_tmy_csv(src)
-    elif outputformat == 'basic':
-        with io.BytesIO(res.content) as src:
-            data, months_selected, inputs, meta = _parse_pvgis_tmy_basic(src)
```

Comment:
What do you think about raising a nicer error message here?  For example `outputformat='basic' is no longer supported, please use outputformat='csv' instead`.

Reply:
Yes, it's been implemented.

Reply:
I believe it was implemented it in `read_pvgis_hourly()` instead of here in `get_pvgis_tmy()`